src/parser.ts

```diff
@@ -161,6 +165,6 @@ export function getNormalizedName(name: string): string {
 export function getApiName(
   apiModel: model.document.BaseUnitWithEncodesModel
```

Comment:
same.. why is apiModel not a model.document.Document?

Reply:
Because it doesn't have to be. This function never uses `declares`, so we don't want to use a type that requires that property.